docs/src/components/landing/Demo.js

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import dynamic from 'next/dynamic';
+
+const DynamicVideo = dynamic(() => import('./DemoVideo'), {
```

Comment:
What's the use case for lazy loading this component? `DemoVideo` is mostly empty.

Reply:
This was intended to help in the performance by having a `fallback` image and then lazy load the `video`; it helps (marginally) when I compared the performance after removing the dynamic component: https://www.webpagetest.org/result/220824_AiDc00_3WF/

Reply:
Wouldn't this cause layout shift? Perhaps we can look into [lazy loading techniques](https://web.dev/lazy-loading-video/#video-gif-replacement) that preserve the markup?

Reply:
I'm not even sure we need to look into lazy loading. The video is one of the first element that we want people to see.

Reply:
> Do we even need lazy loading? The video is one of the first element that we want people to see.

I can see the value if it improves the FCP, we can still autoplay it. Returning visitors may care less about the video than about the rest of the content.

Reply:
Agreed; better initial loading performance without the video is perhaps meaningless. We can remove the `dynamic` . There is already a `poster` attribute on the `video` element, I think adding a default minimum width to it will also fix the 🎄 intermediary state problem.

Reply:
The `poster` could help with video indexation: https://developers.google.com/search/docs/advanced/guidelines/video#provide-a-high-quality-thumbnail